File: Day_25/Q100.cpp
#include<bits/stdc++.h>
using namespace std;

//Comparator Function
bool cmp(string a,string b){
    return a.length() < b.length();
}

//Function To Sort Words By Length
void sort_by_length(vector<string> &arr){

    //Sort Words
    sort(arr.begin(),arr.end(),cmp);
}

int main(){
    int n;
    cin >> n;

    vector<string> arr(n);

    for(int i = 0;i < n;i++){
        cin >> arr[i];
    }

    sort_by_length(arr);

    //Print Result
    for(int i = 0;i < n;i++){
        cout << arr[i] << endl;
    }

    return 0;
}

File: Day_25/Q100.java
import java.util.*;

class Q100{
    static Scanner sc = new Scanner(System.in);

    //Function To Sort Words By Length
    static void sort_by_length(String arr[]){

        //Sort Words
        Arrays.sort(arr,(a,b) -> a.length() - b.length());
    }

    public static void main(String args[]){
        int n = sc.nextInt();

        String arr[] = new String[n];

        for(int i = 0;i < n;i++){
            arr[i] = sc.next();
        }

        sort_by_length(arr);

        //Print Result
        for(int i = 0;i < n;i++){
            System.out.println(arr[i]);
        }

        sc.close();
    }
}